Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2626,7 +2626,13 @@ public IEnumerable<BaseItem> FindExtras(BaseItem owner, IReadOnlyList<FileSystem
                 var current = fileSystemChildren[i];
                 if (current.IsDirectory && _namingOptions.AllExtrasTypesFolderNames.ContainsKey(current.Name))
                 {
-                    var filesInSubFolder = _fileSystem.GetFiles(current.FullName, null, false, false);
+                    var filesInSubFolder = _fileSystem.GetFiles(current.FullName, null, false, false).ToList();
+
+                    if (filesInSubFolder.Any(e => e.Name.Equals(".ignore", StringComparison.Ordinal)))
```

Comment:
Hard coding it seems less than ideal. We already have a resolver that does something similar. Can't we use that here?

Reply:
Do you mean the `ShouldResolvePathContents`? This one expects an `ItemResolveArgs` argument which is tricky to create correctly in the `FindExtras`. That is, if I remember that correctly.

Reply:
I do fully agree with hardcoding the ignore file being less than ideal.